public/js/directives/course/course-edit-modal.html

```diff
@@ -3,10 +3,30 @@
         <h3 class="modal-title">Edit Course</h3>
     </div>
     <div class="modal-body">
-        <labeled-text name="'Name'" model="c.name" style="flex:1 0 100%"></labeled-text>
-        <labeled-text name="'Department'" model="c.dept" style="flex:1"></labeled-text>
-        <labeled-text name="'Number'" model="c.num" style="flex:1"></labeled-text>
-        <labeled-text name="'Credits'" model="c.credits" style="flex:0 0 50px"></labeled-text>
+        <div class="main-course-info">
+            <labeled-text name="'Name'" model="c.name" style="flex:1 0 100%"></labeled-text>
+            <labeled-text name="'Department'" model="c.dept" style="flex:1"></labeled-text>
+            <labeled-text name="'Number'" model="c.num" style="flex:1"></labeled-text>
+            <labeled-text name="'Credits'" model="c.credits" style="flex:0 0 50px"></labeled-text>
+        </div>
+
+        <label for="prereq-dept">Prerequisites</label>
```

Comment:
I see that you have this form in the .html. But I actually don't see this showing up in the edit course modal.